packages/react/src/experimental/OneDataCollection/__stories__/actions.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import { Meta, StoryObj } from "@storybook/react"
-import { Ai, Download, Pencil } from "../../icons/app"
-import { SecondaryActionsDefinition } from "./actions"
-import { OneDataCollection, useDataSource } from "./index"
+import { Ai, Download, Pencil } from "../../../icons/app"
```

Comment:
Probably it's worth importing it this way at this point

```suggestion
import { Ai, Download, Pencil } from "@/icons/app"
```